### PR TITLE
[CS2113T-T09-1]-yzia2000-RecommendedSchedule Feature and checkstyle

### PR DESCRIPTION
**Pull request by yzia2000**

### CODE REVIEW

File: src/main/java/seedu/hustler/command/schedulecommands/AddEntry.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import seedu.hustler.command.Command;
+import seedu.hustler.schedule.RecommendedSchedule;
+import seedu.hustler.ui.Ui;
+
+/**
+ * Adds an entry to recommended schedule.
+ */
+public class AddEntry extends Command {
+
```
Comment (ngjiewu) [inline src/main/java/seedu/hustler/command/schedulecommands/AddEntry.java:11]: Extra line

File: src/main/java/seedu/hustler/schedule/RecommendedSchedule.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+import seedu.hustler.Hustler;
+import seedu.hustler.ui.Ui;
+import java.util.ArrayList;
+
+/**
+ * Extends on the Scheduler in order to provide recommendations
+ * for the task completion mode.
+ */
+public class RecommendedSchedule extends Scheduler {
+
```
Comment (ngjiewu) [inline src/main/java/seedu/hustler/schedule/RecommendedSchedule.java:13]: Extra line

File: src/main/java/seedu/hustler/schedule/ScheduleEntry.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+import seedu.hustler.task.Task;
+
+/**
+ * Entry of the schedule which symbolizes an
+ * incomplete task, amount of time spent on the task,
+ * amount of time allocated by the completion engine and
+ * priority score of the task.
+ */
+public class ScheduleEntry {
+
```
Comment (ngjiewu) [inline src/main/java/seedu/hustler/schedule/ScheduleEntry.java:12]: Extra line

File: src/main/java/seedu/hustler/schedule/Scheduler.java [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+    }
+    
+    /**
+     * Removes entry at an index.
+     *
+     * @param index index at which entry should be removed
+     */
+    public static void remove(int index) {
+        schedule.remove(index);
+
```
Comment (ngjiewu) [inline src/main/java/seedu/hustler/schedule/Scheduler.java:95]: Extra line

File: src/main/java/seedu/hustler/task/Task.java [hunk 5]
```diff
@@ -149,6 +149,15 @@ public boolean equals(Task temp) {
         return this.description.equals(temp.description);
     }
 
+    /**
+     * Checks whether the task has been completed.
+     *
+     * @return task completed or not
+     */
+    public boolean isCompleted() {
```
Comment (ngjiewu) [inline src/main/java/seedu/hustler/task/Task.java:157]: Maybe can name it to more conventional naming for getter method getIsDone?

**Review by ngjiewu — APPROVED** [review]
Good job in adding recommended schedule and cleaning up the codes to match checkstyle.

**Review by nystera — APPROVED** [review]

**Pull request merged**